app/serializers/serialize_solution_for_cli.rb

```diff
@@ -79,4 +78,12 @@ def submission
   def track
     @track ||= solution.exercise.track
   end
+
+  def discussion
+    @discussion = Mentor::Discussion.find_by(mentor: requester, solution: solution)
+  end
```

Comment:
I used the same memoization pattern used in the other methods.

Reply:
I'd probably just replace this and the others with `memoize`. The LOC above doesn't actually memoize - it just resets it every time.